omnibus/files/private-chef-cookbooks/private-chef/libraries/preflight_bootstrap.validator.rb

```diff
@@ -0,0 +1,238 @@
+#
```

Comment:
This file is named a bit oddly.  Should it be `preflight_bootstrap_validator.rb`?